### DISCUSSION TITLE
Item Charges - Potion Doses

### DISCUSSION THREAD

**Discussion by Hoggoh** [runelite/runelite#18314] [Ideas]
If possible add "extended anti-venom+" and "menaphire remedy" to the list of potions that displays their doses.

**Comment by ghost** [comment]
+1 Need to review any others missing as well but definitely these potions

**Comment by YvesW** [comment]
Added in https://github.com/runelite/runelite/commit/f8dd6f5fefbdf1360f534f08451e7c50c16a8401
